eo-runtime/src/main/java/EOorg/EOeolang/EOio/EOstdin$EOnext_line.java

```diff
@@ -59,7 +61,22 @@ public class EOstdin$EOnext_line extends PhDefault {
                                 "There is no line in the standard input stream to consume"
                             );
                         }
-                        return new Data.ToPhi(sc.nextLine());
+
+                        final String firstline = sc.nextLine();
```

Comment:
@includealex Maybe it would be better to rename this variable? Please, read this: https://www.yegor256.com/2015/01/12/compound-name-is-code-smell.html